themes/bootstrap3/js/record.js

```diff
@@ -289,6 +289,24 @@ function applyRecordTabHash() {
 
 $(window).on('hashchange', applyRecordTabHash);
 
+function moreLessSubjects() {
+  $('.more-subjects, .less-subjects').off('click');
```

Comment:
Is there a strong reason to tie this specifically to subjects? Would it be better to use a more generic class so this could be replicated to other fields without requiring additional JS code?

Reply:
Since we use locally a truncate-field class for several fields, I'd support making this more general. We also use a data-rows attribute to optionally indicate the number of lines to show by default. I'm not implying these should be the attributes to use, but something like it might allow us to move to this mechanism, which would be great!